ee/modules/380-metallb/templates/controller/rbac-for-us.yaml

```diff
@@ -13,27 +13,59 @@ metadata:
   {{- include "helm_lib_module_labels" (list . (dict "app" "metallb")) | nindent 2 }}
 rules:
   - apiGroups:
-      - ''
+      - ""
     resources:
       - services
     verbs:
       - get
       - list
       - watch
-      - update
   - apiGroups:
-      - ''
+      - ""
     resources:
       - services/status
     verbs:
       - update
   - apiGroups:
-      - ''
+      - ""
     resources:
       - events
     verbs:
       - create
       - patch
+  - apiGroups:
+      - policy
+    resourceNames:
+      - controller
+    resources:
+      - podsecuritypolicies
+    verbs:
+      - use
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - validatingwebhookconfigurations
+      - mutatingwebhookconfigurations
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
```

Comment:
Why does controller need access for creating crds? And mwh/vwh?

Reply:
removed